README.md

```diff
@@ -1,4 +1,6 @@
 
 # python manage.py runserver
 
-# python manage.py migrate
\ No newline at end of file
+# python manage.py migrate
+
+#DOUBT: tests.py -> 'href="{0}"'.format(homepage_url) what does "{0}" mean?
```

Comment:
{0} is the placeholder for the first variable (hence 0, for variable 0) that goes in the format function. In that case, you only have one variable: `homepage_url`. 

If `homepage_url`="https://pipetilla.com" then the result would be:
'href="https://pipetilla.com"'

That sintax is also equivalent to:
```
'href="{}"'.format(homepage_url)
```
or
```
'href="{homepage_url}"'.format({'homepage_url': homepage_url})
```

From Python 3.6 onwards, you can use also f-strings:
```
f'href="{homepage_url}"'
```

There is more interesting info here:
https://realpython.com/python-f-strings/